app/models/supplejack_api/top_collection_metric.rb

```diff
@@ -96,6 +96,9 @@ def self.find_or_create_top_collection_metric(date, metric, display_collection)
     end
 
     def self.record_metrics_to_be_processed(date, metric, display_collection)
+      # rubocop:disable Metrics/LineLength
+      Rails.logger.info("TOP COLLECTION METRIC: Gathering top 200 records to be processed #{date}, #{metric}, #{display_collection}")
+      # rubocop:enable Metrics/LineLength
```

Comment:
Try to avoid the rubocop disable here :) There's different ways to do it like using `+` or `%W[...].join`

Reply:
good call!